Jenkinsfile

```diff
@@ -49,6 +49,11 @@ pipeline {
 
               echo "Docker tag: ${env.DOCKER_TAG}"
 
+              // Load docker from tar file
+              sh "docker load -i tmail-backend/apps/distributed/target/jib-image.tar"
+              sh "docker load -i tmail-backend/apps/distributed-es6-backport/target/jib-image.tar"
+              sh "docker load -i tmail-backend/apps/memory/target/jib-image.tar"
```

Comment:
better check that this works correctly if it gets merged

Reply:
Fixed